test/spec/controllers/courtSearchInfo.ctrl.spec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+describe('CourtSearchInfoCtrl', function() {
+  var CourtSearchInfoCtrl;
+  var sample_court;
+
+  sample_court = {
+    id: 9,
+    latitutde: 38.7947506,
+    longitude: -90.26261696,
+    municipality: "Black Jack",
+    address: "12500 Old Jamestown Road",
+    city: "St. Louis",
+    state: "MO",
+    zip_code: 63033
+  };
+
+  beforeEach(function() {
+    module('yourStlCourts');
+
+    inject(function($controller,$state, $window, $q, $httpBackend){
+
+      CourtSearchInfoCtrl = $controller('CourtSearchInfoCtrl',{
+        $state: $state,
+        $window: $window,
+        court:sample_court
+      });
+
+      $httpBackend.whenGET(/courts/).respond(200, '');
+    });
+  });
+
+  xit('goes back to home page if court is null',inject(function($state){
```

Comment:
OK, thanks.  I didn't know if I needed to do a whole separate describe and beforeeach...good to know that I don't need that.